miso-web/src/main/webapp/scripts/hot_library.js

```diff
@@ -155,7 +162,17 @@ HotTarget.library = (function() {
         return acc || type.id == lib.libraryTypeId && type.alias.indexOf('Pair') != -1;
       }, lib.paired);
     },
-
+    onLoad: function(config, table) {
+      if (config.sort) {
+        var sortOption = sortOptions[config.sort];
+        var sortColIndex = table.propToCol(sortOption.sortColumn);
+        var rowCount = table.countRows();
+        for (var i = 0; i < rowCount; i++) {
+          table.setCellMeta(i, sortColIndex, 'sortFunction', sortOption.sortFunction);
+        }
+        table.sort(sortColIndex);
```

Comment:
Could extract the code in `hot.js#396-402` into a function and use that here as well...